apps/full-site-editing/full-site-editing-plugin/blocks/page-content/index.js

```diff
@@ -0,0 +1,123 @@
+/* eslint-disable wpcalypso/jsx-classname-namespace */
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+import { find, get, map } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { BlockControls } from '@wordpress/block-editor';
+import { registerBlockType } from '@wordpress/blocks';
+import { IconButton, Placeholder, SelectControl, Toolbar } from '@wordpress/components';
+import { compose, withState } from '@wordpress/compose';
+import { withSelect } from '@wordpress/data';
+import { Fragment, RawHTML } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
```

Comment:
Note that these translation functions won't work out of the box on wpcom without some extra setup; I'd get in touch with Team global about it.

Reply:
I know, and I was wondering how to handle the i18n.
Would it make sense to use `i18n-calypso` instead?

Reply:
I'd assume it depends where these scripts are loaded (Jetpack/wpcom/Calypso) and how that works with files that are loaded from CDN URLs like `widgets.wp.com`, if you use those. i18n gets complicated really fast. :-)

@yoavf is working on making wp.i18n work for Jetpack blocks on wpcom — you might be able to piggyback on that work. At any rate, I'd check with Team global.

I don't think you should worry about it for this PR tho. It's just something I recommend to start figuring out early on.